### ISSUE TITLE
Cannot reuse WinBle after Dispose

### ISSUE THREAD

**Issue by dirablue** [rohitsangwan01/win_ble#10]
When I call dispone of WinBle, cannot reuse this object even I called Initialize again.
could you check this?

```
[ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: Bad state: Cannot add new events after calling close
#0      _BroadcastStreamController.add (dart:async/broadcast_stream_controller.dart:243:24)
#1      WinBle._processMessage
package:win_ble/src/win_ble.dart:89
#2      WinBle.initialize.<anonymous closure>
package:win_ble/src/win_ble.dart:56
#3      _rootRunUnary (dart:async/zone.dart:1434:47)
```

**Comment by rohitsangwan01** [comment]
@dirablue yes some part missing for reinitialising streams after calling dispose , will fix this 
till then i would suggest , don't use dispose

**Comment by dirablue** [comment]
@rohitsangwan01 
got it. thank you

**Comment by rohitsangwan01** [comment]
fixed in release 1.0.0
